### ISSUE TITLE
Add notifications

### ISSUE THREAD

**Issue by npavliuchenko** [npavliuchenko/blackout#8]
dropdown: no, 0, 5min, 15min, 30min

See:
- https://web.dev/push-notifications-overview/
- https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API/Using_the_Notifications_API

**Comment by npavliuchenko** [comment]
BTW how to keep it working on background?

- https://stackoverflow.com/questions/60987496/how-to-trigger-offline-background-function-when-my-pwa-is-closed
- https://stackoverflow.com/questions/56432441/is-there-a-way-to-schedule-offline-notifications-in-a-pwa
- https://stackoverflow.com/questions/70611006/can-a-pwa-schedule-notifications-when-closed
- https://bugs.chromium.org/p/chromium/issues/detail?id=889077

To summarize: OS puts the service worker to sleep, so it can not run background tasks properly. The available options are 'pushing' app from server (which I'd like to avoid) or 'background sync' which can not guarantee the exact timing. Scheduled notifications API was in development a few years ago but canceled. Scheduled Task API is on hold as well. So pushing seems the only solution for now.
